forge-gui/res/cardsfolder/upcoming/goldwardens_gambit.txt

```diff
@@ -0,0 +1,10 @@
+Name:Goldwardens' Gambit
+ManaCost:6 R R
+Types:Sorcery
+K:Affinity:Equipment
+A:SP$ Token | TokenScript$ r_2_2_rebel | TokenOwner$ You | RememberTokens$ True | TokenAmount$ 5 | SubAbility$ DBRepeat | PumpKeywords$ Haste | PumpDuration$ EOT | SpellDescription$ Create five 2/2 red Rebel creature tokens. They gain haste until end of turn. For each of those tokens, you may attach an Equipment you control to it.
+SVar:DBRepeat:DB$ RepeatEach | DefinedCards$ DirectRemembered | RepeatSubAbility$ DBAttach
+SVar:DBAttach:DB$ Attach | Object$ Equipment.YouCtrl | Defined$ Remembered | Optional$ True
```

Comment:
Object$ is Defined syntax

so you probably need to put "Valid " in front of it

Reply:
Ok I made it work with choices though the problem now is that "remembered" contains all the tokens and not just one.